### PR TITLE
Refactor startup command handling

### PR DESCRIPTION
**Pull request by msm-cert**
TODO for the nexts PR: completely eliminate the concept of the "workdir" and "outdir".

* workdir is unnecessary
* outdir should be a regular target directory (temporary directory for karton), cleaned after analysis

### CODE REVIEW

File: drakrun/drakrun/lib/sample_startup.py [hunk 1]
```diff
@@ -39,14 +34,13 @@ def get_office_file_startup_command(extension, file_path):
...
-        return None
+        raise RuntimeError(f"Unknown office file extension {extension}.")
     start_command.extend(["/t", "%f"])
 
     if file_type_allows_macros(extension):
-        vbaparser = VBA_Parser(file_path)
+        vbaparser = VBA_Parser(f"malware.{extension}", data=content)
         if vbaparser.detect_vba_macros():
-            outer_macros = get_outer_nodes_from_vba_file(file_path)
+            outer_macros = get_outer_nodes_from_vba_file(vbaparser)
```
Comment (psrok1) [inline drakrun/drakrun/lib/sample_startup.py:43]: Is it receiving VBA_Parser instance or file path? I think that changes in vba_graph.py might be missing in this PR.

File: drakrun/drakrun/main.py [hunk 2]
```diff
@@ -716,30 +714,22 @@ def process(self, task: Task) -> None:
...
-            self.log.error(
-                "Unable to run malware sample. Could not generate any suitable command to run it."
-            )
-            return
-        if "%f" not in start_command:
-            self.log.warning("No file name in start command")
-
         # If task contains 'custom_hooks' override local defaults
-        with open(os.path.join(workdir, "hooks.txt"), "wb") as hooks:
+        hooks_path = os.path.join(workdir, "hooks.txt")
```
Comment (psrok1) [inline drakrun/drakrun/main.py:731]: Good change :+1:

File: drakrun/drakrun/main.py [hunk 3]
```diff
@@ -592,7 +591,7 @@ def log_startup_failure(self, log_path: str) -> None:
     def analyze_sample(
         self,
         sample_path: str,
-        workdir: str,
+        hooks_path: str,
```
Comment (psrok1) [inline drakrun/drakrun/main.py:594]: Good change :+1:

**Comment by msm-cert** [comment]
```
>>> get_office_file_startup_command("docm", data)
'cmd.exe /C start winword.exe /t %f /mOXDQ /mWorkbook_BeforeClose /mVerifyPath'
```

**Review by psrok1 — APPROVED** [review]

**Pull request merged**